cognite/client/_api/sequences.py

```diff
@@ -1062,8 +1065,7 @@ def _fetch_sequence(post_obj: dict[str, Any]) -> SequenceData:
         tasks_summary = utils._concurrency.execute_tasks(
             _fetch_sequence, [(x,) for x in post_objs], max_workers=self._config.max_workers
         )
-        if tasks_summary.exceptions:
-            raise tasks_summary.exceptions[0]
+        tasks_summary.reraise_if_any_task_failed()
```

Comment:
Added this method as else I don't know which exception type to put in the Raises-section... 😬